ec-europa/eubfr-data-lake
# feat(etl): add poc for wifi4eu - EUBFR-153

To test:
- Add a new set of producer credentials to your `config.json` for `wifi4eu`, pinned in slack
- Run `yarn deploy` to deploy all necessary AWS assets.
- Upload sample data for wifi4eu, file is called `wifi4euRegistrations.xlsx` [EUBFR-86](https://webgate.ec.europa.eu/CITnet/jira/browse/EUBFR-86) as usual
- Use new demo dashboard `http://eubfr-{config.stage}-demo-dashboard-client-wifi4eu.s3-website.eu-central-1.amazonaws.com/` to upload sample data
- Verify ingested data at `https://search-dev-public-g7cnsnvyupyfaccjhtcdr23jai.eu-central-1.es.amazonaws.com/{config.stage}-projects/_search`

Example map of projects: https://codepen.io/kalinchernev/full/xWzegz/
